packages/CollabCodeEdit-Client.package/CCEModel.class/instance/buildWith..st

```diff
@@ -1,9 +1,8 @@
 building
 buildWith: aBuilder
-
 	^ aBuilder build: (aBuilder pluggableWindowSpec new
-		model: self;
-		label: CCEModel defaultApplicationLabel;
-		extent: CCEModel defaultApplicationExtent;
-		children: (self buildChildrenWith: aBuilder);
-		yourself)
\ No newline at end of file
+			model: self;
```

Comment:
Why the additional indentation?

Reply:
With the Squeak indentation it looks ugly otherwise